README.md

```diff
@@ -19,16 +29,23 @@ Refer to [Github releases](https://github.com/UN-OCHA/common_design/releases) fo
 * Variables for breakpoints, colours, font sizes, fonts, and z-index
 * Mixins for clearfix and media queries
 * Custom javascript for dropdown behaviour
-* See [v3.0.0 Release notes](https://github.com/UN-OCHA/common_design/releases/tag/v3.0.0) and [common_design implementation notes for v2](https://docs.google.com/document/d/1GpTtCWNQvGiPDfZmhFvaKGvU9hbOG0HedFTYgo3nvd4) for more detailed documentation
+* See [v3.0.0 Release notes](https://github.com/UN-OCHA/common_design/releases/tag/v3.0.0) and 
+[common_design implementation notes for v2](https://docs.google.com/document/d/1GpTtCWNQvGiPDfZmhFvaKGvU9hbOG0HedFTYgo3nvd4) 
+for more detailed documentation
 
 There is a sub theme included in this repo.
-   * This can be used as a starting point for implementations. Add components, override and extend the base theme as needed. Copy the common_design_subtheme directory to /themes/custom/ and refer to its [README](https://github.com/UN-OCHA/common_design/blob/main/common_design_subtheme/README.md). All the implementation-specific work should happen in the sub theme. Optionally rename the subtheme folder and associated theme files.
+   * This can be used as a starting point for implementations. Add components, override and extend the base theme as 
+   needed. Copy the common_design_subtheme directory to /themes/custom/ and refer to its [README](https://github.com/UN-OCHA/common_design/blob/main/common_design_subtheme/README.md). 
+   All the implementation-specific work should happen in the sub theme. Optionally rename the subtheme folder and 
```

Comment:
The first time I read I could optionally rename the subtheme folder, I guessed this was the recommended way and renamed it (to 'pocam_common_design'). I can't remember the details, but none of the other sites have renamed it, so I ended up renaming it back to 'common_design_subtheme'.

Unless there's a reason to suggest renaming the subtheme, removing this sentence would be clearer.

(see also line 72 below).